scripts/sass-closure-rewriter.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
General comment here is that some people may not understand why we need to rewrite our _sass_ for closure. Should we maybe mention that this is to help make it closure-stylesheets compatible for internal infrastructure?